### PR TITLE
project setup implemented

### PR DESCRIPTION
**Pull request by Ayomikuolatunji**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,6 @@
   "keywords": [
     "gatsby"
   ],
-  "engines": {
```
Comment (hhimanshu) [inline package.json]: Why have we deleted `Node 18` engine?

File: gatsby-node.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
-import { createPages } from "./src/gatsby/createPages";
```
Comment (hhimanshu) [inline gatsby-node.ts]: Why are these commented out?

File: src/Features/Home/DesktopHome/DesktopHome.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+
+export default function DesktopHome() {
+  // console.log("Desktop executing")
```
Comment (hhimanshu) [inline src/Features/Home/DesktopHome/DesktopHome.tsx]: Remove `console.log`

**Review by hhimanshu — COMMENTED** [review]
1. Delete `.DS_Store` file and add it to `.gitignore`. This file can exists recursively in any directly so use glob pattern (`*.DS_Store`)

File: src/Features/Home/DesktopHome/DesktopHome.tsx (hunk 3 above)
Comment (Ayomikuolatunji) [reply src/Features/Home/DesktopHome/DesktopHome.tsx]: Done

File: gatsby-node.ts (hunk 2 above)
Comment (Ayomikuolatunji) [reply gatsby-node.ts]: It was intentional. I have  added it back.

File: package.json (hunk 1 above)
Comment (Ayomikuolatunji) [reply package.json]: I will add back

**Review by hhimanshu — APPROVED** [review]

**Pull request merged**